lib/config.js

```diff
@@ -0,0 +1,62 @@
+/*
+ Copyright 2017 Bitnami.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+*/
+
+'use strict';
+
+const _ = require('lodash');
+const helpers = require('./helpers');
+const request = require('request');
+
+class Config {
+  constructor(options) {
+    const opts = _.defaults({}, options, {
+      namespace: 'kubeless',
+    });
+    this.namespace = opts.namespace;
+    const APIRootUrl = helpers.getKubernetesAPIURL(helpers.loadKubeConfig());
+    const url = `${APIRootUrl}/api/v1/namespaces/${opts.namespace}/configmaps/kubeless-config`;
+    this.connectionOptions = Object.assign(
```

Comment:
I would be careful with this. this is assuming that every user will have read access on that configmap. So we need to check the RBAC settings for it, otherwise this could break.

Reply:
Yes. The configmap should readable by default. I tried this with RBAC enabled and it works.